webapp/src/jsTest/kotlin/serialization/SerializerTest.kt

```diff
@@ -1,38 +0,0 @@
-@file:UseSerializers(
-    OptionSerializer::class,
-)
-
-package com.juul.koap.serialization
-
-import com.juul.koap.Message.Option
-import com.juul.koap.Message.Option.Accept
-import com.juul.koap.Message.Option.ContentFormat
-import com.juul.koap.Message.Option.ETag
-import com.juul.koap.Message.Option.UriHost
-import com.juul.koap.Message.Option.UriPort
-import kotlinx.serialization.UseSerializers
-import kotlinx.serialization.json.Json
-import kotlin.test.assertEquals
-import kotlin.test.Test
-
-class SerializerTest {
-    @Test
-    fun optionSerializesTo() {
-        assertOptionSerializesTo(ContentFormat.PlainText, "Content-Format: text/plain; charset=utf-8")
```

Comment:
moving serializer test to #341 only, where plenty of more special serializations are added